Binary files StockQuoteService3/.DS_Store and StockQuoteService2/.DS_Store differ
diff -r StockQuoteService3/.project StockQuoteService2/.project
3c3
< 	<name>StockQuoteService3</name>
---
> 	<name>StockQuoteService2</name>
diff -r StockQuoteService3/AndroidManifest.xml StockQuoteService2/AndroidManifest.xml
3c3
<       package="com.androidbook.services.stock3"
---
>       package="com.androidbook.services.stock2"
6,10c6,8
<     <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19" />
<     <application android:icon="@drawable/icon"
<             android:label="@string/app_name">
<         <activity android:name=".MainActivity"
<                 android:label="@string/app_name"
---
>     <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19"/>
>     <application android:icon="@drawable/icon" android:label="@string/app_name">
>         <activity android:name=".MainActivity"  android:label="@string/app_name"
16c14
<       <service android:name="StockQuoteService3">
---
>       <service android:name="StockQuoteService2">
18c16
<             <action android:name="com.androidbook.services.stock3.IStockQuoteService" />
---
>             <action android:name="com.androidbook.services.stock2.IStockQuoteService" />
22d19
< 
Only in StockQuoteService2/gen: .DS_Store
Only in StockQuoteService2/gen/com: .DS_Store
Only in StockQuoteService2/gen/com/androidbook: .DS_Store
Only in StockQuoteService2/gen/com/androidbook/services: .DS_Store
Only in StockQuoteService2/gen/com/androidbook/services: stock2
Only in StockQuoteService3/gen/com/androidbook/services: stock3
Only in StockQuoteService2/res: drawable
Only in StockQuoteService3/res/drawable-mdpi: emo_im_happy.png
diff -r StockQuoteService3/res/layout/main.xml StockQuoteService2/res/layout/main.xml
5,6c5
<     android:layout_width="fill_parent"
<     android:layout_height="fill_parent" >
---
>     android:layout_width="fill_parent"  android:layout_height="fill_parent" >
8,10c7,8
<     android:layout_width="fill_parent"
<     android:layout_height="wrap_content"
<     android:text="This is where the service could ask for help." />
---
>     android:layout_width="fill_parent"  android:layout_height="wrap_content" 
>     android:text="This is where the service could ask for help, or present controls." />
12d9
< 
diff -r StockQuoteService3/res/values/strings.xml StockQuoteService2/res/values/strings.xml
3,4c3
<     <string name="hello">Hello World, MainActivity!</string>
<     <string name="app_name">StockQuoteService3</string>
---
>     <string name="app_name">Stock Quote Service2</string>
Binary files StockQuoteService3/src/.DS_Store and StockQuoteService2/src/.DS_Store differ
Binary files StockQuoteService3/src/com/.DS_Store and StockQuoteService2/src/com/.DS_Store differ
Binary files StockQuoteService3/src/com/androidbook/.DS_Store and StockQuoteService2/src/com/androidbook/.DS_Store differ
Binary files StockQuoteService3/src/com/androidbook/services/.DS_Store and StockQuoteService2/src/com/androidbook/services/.DS_Store differ
Only in StockQuoteService3/src/com/androidbook/services/stock2: .DS_Store
diff -r StockQuoteService3/src/com/androidbook/services/stock2/IStockQuoteService.aidl StockQuoteService2/src/com/androidbook/services/stock2/IStockQuoteService.aidl
2,3c2,3
< package com.androidbook.services.stock3;
< import com.androidbook.services.stock3.Person;
---
> package com.androidbook.services.stock2;
> import com.androidbook.services.stock2.Person;
7c7
<     String getQuote(in String ticker,in Person requester);
---
>     String getQuote(String ticker,in Person requester);
diff -r StockQuoteService3/src/com/androidbook/services/stock2/MainActivity.java StockQuoteService2/src/com/androidbook/services/stock2/MainActivity.java
1,13c1,13
< package com.androidbook.services.stock3;
< 
< import android.app.Activity;
< import android.os.Bundle;
< 
< public class MainActivity extends Activity {
<     /** Called when the activity is first created. */
<     @Override
<     public void onCreate(Bundle savedInstanceState) {
<         super.onCreate(savedInstanceState);
<         setContentView(R.layout.main);
<     }
< }
\ No newline at end of file
---
> package com.androidbook.services.stock2;
> 
> import android.app.Activity;
> import android.os.Bundle;
> 
> public class MainActivity extends Activity {
> 
> 	@Override
> 	protected void onCreate(Bundle savedInstanceState) {
> 		super.onCreate(savedInstanceState);
> 		setContentView(R.layout.main);
> 	}
> }
diff -r StockQuoteService3/src/com/androidbook/services/stock2/Person.aidl StockQuoteService2/src/com/androidbook/services/stock2/Person.aidl
2c2
< package com.androidbook.services.stock3;
---
> package com.androidbook.services.stock2;
diff -r StockQuoteService3/src/com/androidbook/services/stock2/Person.java StockQuoteService2/src/com/androidbook/services/stock2/Person.java
1c1
< package com.androidbook.services.stock3;
---
> package com.androidbook.services.stock2;
3d2
< //This file is Person.java
diff -r StockQuoteService3/src/com/androidbook/services/stock2/StockQuoteService2.java StockQuoteService2/src/com/androidbook/services/stock2/StockQuoteService2.java
1c1,2
< package com.androidbook.services.stock3;
---
> package com.androidbook.services.stock2;
> // This file is StockQuoteService2.java
12c13
< public class StockQuoteService3 extends Service
---
> public class StockQuoteService2 extends Service
20,21c21
<             return "Hello " + requester.getName() +
<                 "! Quote for " + ticker + " is 20.0";
---
>             return "Hello "+requester.getName()+"! Quote for "+ticker+" is 20.0";
32c32
<         displayNotificationMessage("onCreate() called in StockQuoteService3");
---
>         displayNotificationMessage("onCreate() called in StockQuoteService2");
38c38
<         displayNotificationMessage("onDestroy() called in StockQuoteService3");
---
>         displayNotificationMessage("onDestroy() called in StockQuoteService2");
43a44,47
>     public int onStartCommand(Intent intent, int flags, int startId) {
>         return super.onStartCommand(intent, flags, startId);
>     }
> 
47c51
<         displayNotificationMessage("onBind() called in StockQuoteService3");
---
>         displayNotificationMessage("onBind() called in StockQuoteService2");
57c61
<             .setContentTitle("StockQuoteService3")
---
>             .setContentTitle("StockQuoteService2")
65c69
< 
---
>         
